website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md

```diff
@@ -21,8 +21,8 @@ The exact URI to be set here can be obtained from MongoDB Cloud's web interface.
 
 :::note Important
 * Use a MongoDB database with Appsmith, `replicaSet` needs to be configured on the database. This can be done by connecting to the database as `admin`, and running [`rs.initiate()`](https://docs.mongodb.com/manual/reference/method/rs.initiate/). If you are using MongoDB Cloud, this is always done for you.
```

Comment:
> Use a MongoDB database with Appsmith

I guess it should be: To use an existing MongoDB database with Appsmith

Reply:
Updated the text.